examples/v1/taskruns/default_task_params.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  # This has to be explicit instead of `generateName`, since it will be referenced
+  # by the TaskRun
+  name: example-default-task-param
+spec:
+  params:
+    - name: input
+      default: "No input provided, but that's okay!"
```

Comment:
it really iz ok

Reply:
--- FAIL: TestExamples/v1/taskruns/default_task_params (4.26s)
the example seems to be failing in the e2e tests

Reply:
Forgot to enclose `$(params.input)` in quotes so the `!` in the default value caused some shell issues 😅

Reply:
that's alright, looks good to me as of now, shall wait for the CI to pass